### DISCUSSION TITLE
2022/02/10/Mysql%E6%8F%92%E5%85%A5%E5%88%A0%E9%99%A4%E6%AD%BB%E9%94%81%E9%97%AE%E9%A2%98%E6%8E%92%E6%9F%A5/

### DISCUSSION THREAD

**Discussion by giscus** [skitii/skitii#3] [General]
# 2022/02/10/Mysql%E6%8F%92%E5%85%A5%E5%88%A0%E9%99%A4%E6%AD%BB%E9%94%81%E9%97%AE%E9%A2%98%E6%8E%92%E6%9F%A5/

搞懂Mysql插入删除是如何加锁的

http://localhost:4000/2022/02/10/Mysql%E6%8F%92%E5%85%A5%E5%88%A0%E9%99%A4%E6%AD%BB%E9%94%81%E9%97%AE%E9%A2%98%E6%8E%92%E6%9F%A5/

**Comment by AbnerHuang2** [comment]
ddd

**Reply by AbnerHuang2** [reply]
nb
